__tests__/app.test.js

```diff
@@ -165,10 +164,48 @@ describe("GET reviews ", () => {
             owner: expect.any(String),
             review_body: expect.any(String),
             review_img_url: expect.any(String),
-            created_at: expect.any(Number),
+            created_at: expect.any(String),
             votes: expect.any(Number),
+            comment_count: expect.any(Number),
           });
         });
       });
   });
+  it("200: should respond with an array sorted by date,descending", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.review).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  it("200: should be sorted by the correct key when passed a sort query", () => {
+    return request(app)
+      .get("/api/reviews?category=dexterity")
+      .expect(200)
+      .then(({ body }) => {
+        body.review.forEach(() => {
```

Comment:
Maybe add an extra expect statement here, a length test to show us how many reviews were returned with the category of dexterity...... for example, if there are 3 reviews in the database with a category of 'dexterity', do we have 3 back in this api call?